airbyte-integrations/connectors/source-zoho-crm/source_zoho_crm/streams.py

```diff
@@ -0,0 +1,143 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+import concurrent.futures
+import datetime
+import math
+from abc import ABC
+from dataclasses import asdict
+from typing import Any, Dict, Iterable, List, Mapping, MutableMapping, Optional
+
+import requests
+from airbyte_cdk.sources.streams.http import HttpStream
+
+from .api import ZohoAPI
+from .exceptions import IncompleteMetaDataException, UnknownDataTypeException
+from .types import FieldMeta, ModuleMeta, ZohoPickListItem
+
+
+class ZohoCrmStream(HttpStream, ABC):
+    json_schema: Dict[Any, Any] = {}
+    _path: str = ""
+
+    def next_page_token(self, response: requests.Response) -> Optional[Mapping[str, Any]]:
+        if response.status_code != 200:
+            return None
+        pagination = response.json()["info"]
+        if not pagination["more_records"]:
+            return None
+        return {"page": pagination["page"] + 1}
+
+    def request_params(
+        self, stream_state: Mapping[str, Any], stream_slice: Mapping[str, any] = None, next_page_token: Mapping[str, Any] = None
+    ) -> MutableMapping[str, Any]:
+        if next_page_token:
```

Comment:
I think `return {**next_page_token} if next_page_token else {}` will be more pythonic solution. 🙂

Reply:
done